modules/core/src/test/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/IgniteThrottlingUnitTest.java

```diff
@@ -100,34 +100,51 @@ public class IgniteThrottlingUnitTest extends GridCommonAbstractTest {
      *
      */
     @Test
-    public void breakInCaseTooFast() {
+    public void shouldThrottleWhenWritingFasterThanTargetSpeedAndDirtyRatioIsAboveTargetRatio() {
```

Comment:
it's a bit too hard to read without spacing. Maybe leave the old name and add a clarifying comment?

Reply:
Shortened the name